### ISSUE TITLE
Add KilowattHourPerPound (kWh/lbs) to SpecificEnergy

### ISSUE THREAD

**Issue by rutemja** [angularsen/UnitsNet#1090]
**Is your feature request related to a problem? Please describe.**
Our machines need to show the used energy per produced kilogram/pound. UnitsNet already supports the SI units in SpecificEnergy class, but the imperial units (like kWh/lbs) are missing.

**Describe the solution you'd like**
Please add kWh/lbs to SpecificEnergy class.

These should be the correct functions to calculate it from J/kg:
```
"FromUnitToBaseFunc": "{x} * 7.937e+6",
"FromBaseToUnitFunc": "{x} / 7.937e+6",
```

Thank you :)

**Comment by angularsen** [comment]
Hi, would you be interested in attempting a pull request? I'm happy to assist.
Same with #1091 .

https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit

**Comment by rutemja** [comment]
Done (my first PR 🥳)

**Comment by angularsen** [comment]
Great job! 

Fixed in #1092 .
